package.json

```diff
@@ -68,6 +68,7 @@
     "imports-loader": "0.8.0",
     "intl-messageformat": "9.11.4",
     "jest": "27.0.6",
+    "jodit-react": "^4.1.2",
```

Comment:
I would assume draft-js and draft-js-export-html should be removed with this?

Reply:
yes, forgot to remove